packages/airnode-abi/tsconfig.json

```diff
@@ -2,8 +2,8 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "baseUrl": "./",
-    "outDir": "dist"
+    "outDir": "./dist",
```

Comment:
Should we not move all of these options to the base file to make it consistent for all packages?

Reply:
Unfortunately, the `outDir` is relative to the path of the tsconfig. This path is different for every monorepo package so there is no other way then listing it explicitely.